examples/requests_https_cpython.py

```diff
@@ -1,21 +1,35 @@
 # adafruit_requests usage with a CPython socket
```

Comment:
Change to `#adafruit_requests usage with SocketPool`

Reply:
it is impossible for me to tell when an example is for Python on something like a Raspberry pi or whether it's for the Metro ESP32-S2 or the older ESP32 boards.  I just took an example in that directory and found one without the esp32spi library and messed with it until it worked.  I would love to see the structure of these examples broken down into what works on what boards.  I'm just struggling to figure this out.

Reply:
I understand, and that's why we're making this PR, to help differentiate between examples.

CPython refers to Python on a Raspberry Pi or Computer
CircuitPython refers to Python on a Microcontroller like an ESP32-S2.

This example would be named something like `requests_https_socketpool` since it uses the native socket pool  API (https://circuitpython.readthedocs.io/en/latest/shared-bindings/socketpool/index.html#socketpool.SocketPool.socket) for ESP32S2.

Reply:
I think we should use the following suffixes for these examples: 
`_cpython` - Python on Computer/Raspberry Pi
`_esp32spi` - Using ESP32SPI as a Co-Processor
`_socketpool` - Using native CircuitPython sockets

Do you agree with this? Any tips on how I could clarify this more?

Reply:
I did not grasp the many types of pythons that there were.  My example was purely for use on the Metro ESP32-S2 that I have.  I have used some the Cpython on my Raspbeerry Pi.  I will defer to your best judgement on this but comments clarifying all this is good.